assets/js/modules/analytics-4/components/common/PropertySelect.test.js

```diff
@@ -92,48 +89,6 @@ describe( 'PropertySelect', () => {
 		expect( listItems ).toHaveLength( properties.length + 1 );
 	} );
 
-	it( 'should not render if account ID is not valid', async () => {
```

Comment:
See above, I'm not sure why this behavior needs to be removed here? I think it should still be the case. The property dropdown shouldn't display in any way if no account is selected yet.

Reply:
Agree. Reverted.